frontend/__tests__/services/loadChallenges.spec.js

```diff
@@ -27,6 +27,7 @@ describe(loadChallenges, () => {
             const { challenges } = await loadChallenges({
                 typeFilter: mock.filter.undefined,
             });
+            expect(capitalize).not.toHaveBeenCalled();
```

Comment:
Thinking about it.. I think it must be better to serialize the values on the back-end instead.

I'll open an issue related to that.

@stevescruz